src/components/calcite-dropdown-item/calcite-dropdown-item.tsx

```diff
@@ -138,10 +138,10 @@ export class CalciteDropdownItem {
     );
     return (
       <Host
-        aria-selected={this.active.toString()}
+        aria-checked={this.active.toString()}
         dir={dir}
         isLink={this.href}
-        role="menuitem"
+        role="menuitemcheckbox"
```

Comment:
Should this be dependent on type of dropdown group?

Menu item Checkbox for multi select,

Menu item Radio for single select, 

Just menu item for "none" (since they function as links or items?)

Reply:
Yeah maybe. I can remove and create an issue for someone to tackle?

Reply:
Yeah assign to me and remove from this, I think it's probably going to have to change based on dropdown group type.

Reply:
removed!